.github/workflows/nightly-benchmark-overhead.yml

```diff
@@ -9,10 +9,8 @@ jobs:
   run-overhead-tests:
     runs-on: ubuntu-latest
     steps:
-      - name: checkout
-        uses: actions/checkout@v2.3.4
-      - name: checkout
-        uses: actions/checkout@v2.3.4
+      - uses: actions/checkout@v2.3.4
+      - uses: actions/checkout@v2.3.4
```

Comment:
```suggestion
```
looks like a duplicate line

Reply:
the second one is checking out `gh-pages` branch to a subdirectory